pages/movies.js

```diff
@@ -0,0 +1,103 @@
+import * as React from "react";
+import {
+  Card,
+  CardContent,
+  CardMedia,
+  Typography,
+  Button,
+  CardActionArea,
+  CardActions,
+  Grid,
+  ThemeProvider,
+  createTheme,
+} from "@mui/material";
+import dbConnect from "../lib/dbConnect";
+import Movie from "../models/movie";
+
+// Create a theme for this component
+const moviesTheme = createTheme({
+  palette: {
+    primary: {
+      main: '#66aca6',
+    },
+    secondary: {
+      main: '#233f3d',
+    },
+    background: {
+      paper: "rgba(102, 172, 166, 0.1)",
+    },
+  },
+  typography: {
+    fontFamily: ['"Playfair Display"'],
+  },
+});
```

Comment:
Things like this you would usually put in a global theme (e.g. in `_document.js` or `_app.js`) to use the same styles across the entire site.

There are use cases for local overrides (like this one) for sure, but they're rare since most pages would use the same color palette as every other page.